scripts/run-terraform-import.ts

```diff
@@ -23,7 +23,10 @@ class TerraformImportRunner {
     }
     
     try {
-      await this.synthesizeTerraformConfig();
+      if (!this.isPreviewMode) {
+        await this.synthesizeTerraformConfig();
+      }
```

Comment:
Incorrect. We want this in preview mode.

The failure is due to the terraform cli not being found. This script should ensure it's in the GHA runner